src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> As for the second, if all goes well, maybe all we need is a clarification in the docs. Something like, "The native event object is available at event.originalEvent, if the original event is not a simulated event (e.g. "focus")."

But that's never true for any widget events in jQuery UI. A native event *may* exist, but if it does, it's at least two levels deep. For example, `menufocus` could have an original event of `focus` (via `jQuery.Event()`) which itself has an original event of `focus` (via a native event).